pkg/client/linstor.go

```diff
@@ -488,10 +495,24 @@ func (s *Linstor) Attach(ctx context.Context, volId, node string, readOnly bool)
 		}
 
 		s.log.Infof("volume %s does not exist on node %s, creating new resource", volId, node)
+		var controllerQuorum bool
+		controllerProps, err := s.client.Controller.GetProps(ctx)
+		if err != nil {
+			s.log.Infof("No controller properties found")
+			return nil
+		} 
+		for k, v := range controllerProps {
+			if k == "DrbdOptions/auto-quorum" {
+				if v != "disabled" {
+					controllerQuorum = true
+				}
+			}
+		}
+		useQuorum := volQuorum || controllerQuorum
```

Comment:
We don't need to check the controller properties here. 

First of all we would also need to check the resource group property.

Secondly, this setting does not influence _existing_ resources, so only whatever is set directly on the resource definition is relevant.

So: just remove this and use `volQuorum` directly.